docs/Examples.md

```diff
@@ -0,0 +1,107 @@
+In the following sections, we present example configurations for a variety of Git Providers and Issue Trackers combinations.
+
+## Bitbucket & Jira
+
+```yaml
+apiVersion: charts.pelorus.konveyor.io/v1alpha1
+kind: Pelorus
+metadata:
+  name: bitbucket-jira-example-configuration
+spec:
+  exporters:
+    global: {} # this line is necessary?
+    instances:
+      - app_name: deploytime-exporter
+        exporter_type: deploytime
+        extraEnv:
+          - name: NAMESPACES
```

Comment:
I was thinking of having a big file with all configuration set and commented, or do you prefer to have comments in this file ?

Reply:
talked to Wes, we will let the files clean